src/importer.ts

```diff
@@ -167,36 +163,32 @@ export default class SystemImporter implements Importer {
 
     const batchId = parseInt(batchIdMatch[1])
 
-    await this.interpreter.interpretFile({
+    const cvr = await this.interpreter.interpretFile({
       election: this.election,
       ballotImagePath,
-      cvrCallback: ({ ballotImagePath, cvr }: CVRCallbackParams) => {
-        this.cvrCallbackWithBatchId({ batchId, ballotImagePath, cvr })
-      },
```

Comment:
technically I had a thought that maybe we'd eventually have more than one CVR per file in some vague future, but in retrospect that makes no sense, and even then your API is better. Nice.